RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# Releasing
+
+Before releasing a new version of packages to the NPM and GitHub, Its advisable to follow the below steps in order
+
+- Update the dependencies `npm run clean`
+- Bootstrap the deps using `npm run bootstrap`
+- Make sure all tests passes `npm run test`
+
+### Releasing all packages
+
+To publish all the packages run `npm run publish`. It will run lerna publish (we use --independent) so it will prompt the version number for
+every package. `lerna publish` will take care of the following
+
+- Creating annonated Git tags for all the packages that has changed since last release
+- Generating `CHANGELOG.md` using conventional commit messages.
+- Publishing the packages via `npm publish`
+
+### Releasing single package
+
+To publish a single package, run `npm run release-package -- elastic-apm-js-base` which uses `lerna publish` under the hood to publish a single package to NPM
```

Comment:
Is it possible to merge this with the publish script? Maybe something to accept a variable like `scope` to provide the package name.

Reply:
scope in general is tied to `lerna run` command.. Since its our own custom script, We can use scope though. Do you want like that?

Reply:
I don't have a strong preference in regards to the name of the variable, although I do think `scope=elastic-apm-js-base` is more clear than just ` -- elastic-apm-js-base`. 

But my main point was to have one script that handles the release instead of two.

Reply:
Hmmm, the reason i kept it at separate is because of failures. We can do that later once we feel conformable with the release process.